fixtures/git/git_protocol_pass.yaml

```diff
@@ -0,0 +1,30 @@
+#
+# This check will perform pull and push operations to the Git repository over HTTP protocol, targeting the file named 'test.txt'.
+# Prerequisites: A non-empty Git repository that will be available via the HTTP protocol.
+#
+
+apiVersion: canaries.flanksource.com/v1
+kind: Canary
+metadata:
+  name: gitprotocol-check
+  labels:
+    app: gitlab
+    env: production
+    owner: sre
+    stream: sre
+    product: observability
+    temporary: "false"
+    deleteAfter: "false"
+spec:
+  interval: 30
+  gitProtocol:
+    - name: gitlab-check
+      test:
+        expr: size(results) > 0
+      repository: https://gitlab.com/devops/canary-checker-poc.git
```

Comment:
Ideally we want the fixtures to be self contained - which would entail deploying a gitea or gogs git server with a repo and user/pass in the fixtures/git/_setup.yaml file

Reply:
This is quite problematic to do. Can we skip it?